README.md

```diff
@@ -142,3 +142,30 @@ an input and returns a str. By default, `RichHelpFormatter` sets the function to
 ```python
 RichHelpFormatter.group_name_formatter = str.title
 ```
+
+## Contributing
+Start by [forking the repo](https://github.com/hamdanal/rich-argparse/fork) and cloning your fork.
+
+For those using [VS Code](https://code.visualstudio.com/) there's [preconfigured settings](.vscode/) in the repo.
+
+### Environment Setup
+You can use `tox` to setup a virtual environment for `rich_argparse` and install dependencies.
+
+```sh
+pip3 install tox
```

Comment:
Let's delete this line and have the tox docs (to be) linked above point to the preferred way of installing tox